frontend/src/hooks/useSessionValid.js

```diff
@@ -35,16 +33,13 @@ function useSessionValid() {
       if (!orgs?.length) {
         throw Error("Organizations not available.");
       }
-      if (orgs?.length > 1 && !signedInOrgId?.length) {
+      if (orgs?.length > 1 && signedInOrgId && signedInOrgId === "public") {
```

Comment:
@muhammad-ali-e Can we simplify this condition to this,

`orgs?.length > 1 && signedInOrgId === "public"`

Reply:
updated here. I think It's wrong to check public here.